superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -55,7 +55,7 @@ import {
 import Loading from 'src/components/Loading';
 import ExtraOptions from './ExtraOptions';
 import SqlAlchemyForm from './SqlAlchemyForm';
-import DatabaseConnectionForm from './DatabaseConnectionForm';
+import DatabaseConnectionForm from './DatabaseConnectionForm/DatabaseConnectionForm';
```

Comment:
Rename `/DatabaseConnectionForm/DatabaseConnectionForm` to `/DatabaseConnectionForm/index.tsx` so we can just import like this:
```
import DatabaseConnectionForm from './DatabaseConnectionForm/DatabaseConnectionForm';
```